assets/src/design-system/components/snackbar/snackbarContainer.js

```diff
@@ -93,9 +95,9 @@ const ChildContainer = styled.div`
 
 export const SnackbarContainer = ({
   component: Component = SnackbarMessage,
-  notifications,
-  onRemove,
-  placement,
+  notifications = [],
+  onRemove = noop,
```

Comment:
Each notification can have an `onDismiss` prop, so I made this prop not required and defaulted it to `noop`. We may not need to do this 🤷